Components/TestComponents/ZodTestComponent.tsx

```diff
@@ -0,0 +1,42 @@
+import { formSchema , FormDataTypes } from '../../src/schemas'
+import { useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import './ZodTestComponentCss.css';
+import React from 'react';
+
+
+export function ZodTestComponent() {
```

Comment:
Components should be exported as default functions.

Reply:
True